WrightTools/artists/_interact.py

```diff
@@ -14,6 +14,32 @@
 __all__ = ["interact2D"]
 
 
+class Focus:
```

Comment:
Strange use of a class, to be honest.
Something that only implements `__call__` should probably just be a function plus a private variable, especially when the business logic in `__init__` is the same as in `__call__`

That said, I don't think it warrents delaying merging to fix this up, it is not in `__all__` and is thus a private implementation detail.

Reply:
Could also be implemented as a method of `Bunch`, as that is how it is used internally, maybe... Bunch is itself a weird object because it inherits from dict and sets `__dict__ = self`